bentoml/_internal/runner/container.py

```diff
@@ -14,7 +14,7 @@
 
 SingleType = t.TypeVar("SingleType")
 BatchType = t.TypeVar("BatchType")
-IndexType = t.Union[None, int]
+BatchIndicesType = list[int]
```

Comment:
This will cause exception on lower version of Python. Using list/tuple as type is only allowed in type annotation area, or in `if TYPE_CHECKING:` since it's always Python 3.10

Reply:
@larme

Reply:
@bojiang fixed